gdal/swig/python/samples/README

```diff
@@ -4,115 +4,136 @@ Python interface may be used in your applications.
-			fast Fourier transform.
+fft.py
+    Script to perform forward and inverse two-dimensional fast Fourier transform.
 
-gdal2grd.py		Script to write out ASCII GRD rasters (used in Golden
-			Software Surfer) from any source supported by GDAL.
+gdal2grd.py
+    Script to write out ASCII GRD rasters (used in Golden Software Surfer)
+    from any source supported by GDAL.
 
-gdal_vrtmerge.py	Similar to gdal_merge.py, but produces a VRT file.
+gdal_vrtmerge.py
+    Similar to gdal_merge.py, but produces a VRT file.
 
-gdalcopyproj.py		Duplicate the geotransform, projection and/or GCPs from
-			one raster dataset to another, which can be useful after
-			performing image manipulations with other software that
-			ignores or discards georeferencing metadata.
+gdalcopyproj.py
+    Duplicate the geotransform, projection and/or GCPs from	one raster dataset to
+    another, which can be useful after performing image manipulations with other
+    software that ignores or discards georeferencing metadata.
 
-gdalfilter.py		Example script for applying kernel based filters to
-			an image using GDAL.  Demonstrates use of virtual
-			files as an intermediate representation.
+gdalfilter.py
+    Example script for applying kernel based filters to	an image using GDAL.
+    Demonstrates use of virtual	files as an intermediate representation.
 
-gdal_retile.py          Script for restructuring data in a tree of regular tiles.
-                        (transferred in swig/python/scripts)
+gdal_retile.py
+    Script for restructuring data in a tree of regular tiles.
+    (transferred in swig/python/scripts)
 
-get_soundg.py		Script to copy the SOUNDG layer from an S-57 file to
-			a Shapefile, splitting up features with MULTIPOINT
-			geometries into many POINT feature, and appending
-			the point elevations as an attribute.
+get_soundg.py
+    Script to copy the SOUNDG layer from an S-57 file to a Shapefile,
+    splitting up features with MULTIPOINT geometries into many POINT feature,
+    and appending the point elevations as an attribute.
 
-histrep.py		Module to extract data from many rasters into one output.
+histrep.py
+    Module to extract data from many rasters into one output.
 
-load2odbc.py		Load ODBC table to an ODBC datastore.  Uses direct SQL
-			since the ODBC driver is read-only for OGR.
+load2odbc.py
+    Load ODBC table to an ODBC datastore.
+    Uses direct SQL	since the ODBC driver is read-only for OGR.
 
-rel.py			Script to produce a shaded relief image from the
-			elevation data. (similar functionality in gdaldem now)
+rel.py
+    Script to produce a shaded relief image from the elevation data.
+    (similar functionality in gdaldem now)
 
-tigerpoly.py		Script demonstrating how to assemble polygons from
-			arcs in TIGER/Line datasource, writing results to
-			a newly created shapefile.
+tigerpoly.py
+    Script demonstrating how to assemble polygons from
+	arcs in TIGER/Line datasource, writing results to
+	a newly created shapefile.
```

Comment:
Tabs in the beginning of 2nd the 3rd line

Reply:
this one I've missed.